src/test/java/seedu/address/logic/commands/SortCommandTest.java

```diff
@@ -29,7 +29,7 @@ public void setUp() {
      */
     @Test
     public void execute_listIsNotFiltered_showsSortedList() {
-        expectedModel.sortPetList();
-        assertCommandSuccess(new SortCommand(), model, SortCommand.MESSAGE_SUCCESS, expectedModel);
+        expectedModel.sortPetList("o");
+        assertCommandSuccess(new SortCommand("o"), model, SortCommand.MESSAGE_SUCCESS, expectedModel);
```

Comment:
Maybe consider adding additional test cases for sorting by pet name as well?

Reply:
Good idea!